controllers/event.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express.Router();
+
+const bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
It should be express() (your `app` const in app.js) what should use the bodyParser middleware, not the router. It's confusing because, as I pointed above, you are assigning the wrong thing to the `app` variable.